compiler-core/src/language_server/rename.rs

```diff
@@ -65,6 +65,102 @@ pub fn rename_local_variable(
     Some(workspace_edit(uri, edits.edits))
 }
 
+pub fn rename_module_value(
+    params: &RenameParams,
+    module_name: &EcoString,
+    name: &EcoString,
+    modules: &HashMap<EcoString, Module>,
+    name_kind: Named,
+) -> Option<WorkspaceEdit> {
+    if name::check_name_case(
+        Default::default(),
```

Comment:
Could you use the default impl on the type that is being constructed rather than the method please? It makes it easier to understand what is being made when it is explicit.